### PR TITLE
Added alumni section

### PR DESCRIPTION
**Pull request by kumanik5661**
Separate model for alumni created corresponding ViewSet and serializers and API endpoint added.

### CODE REVIEW

**Comment by realsdx** [comment]
Good.
Also add the feature to copy existing data to Alumni table from Profile table.

**Comment by kumanik5661** [comment]
A convert_to_alumni flag is added to Profile model. If a admin user saves the Profile object with the flag as true the object will be deleted and a corresponding Alumni model object will be created

**Comment by realsdx** [comment]
Alumni shouldn't be associated with an user. At least not to an OneToOne field. If you want to add user attribute to the Alumni, at least make it nullable.

**Comment by realsdx** [comment]
Good work. It's nice that you added the batch converting options.
But, with Django you don't need to these heavy lifting (creating pages and all).
Just add an `Action` in Alumni admin panel. Django will do it all.

**Comment by kumanik5661** [comment]
Did that

**Comment by realsdx** [comment]
Great! Works as expected. One last thing, If a user exists for the Profile(currently it's required), then make `user.active=False` for that user when converting to alumni.

File: main/models.py [hunk 1]
```diff
@@ -124,6 +124,9 @@ def save(self, *args, **kwargs):
         """
         if self.convert_to_alumni == True:
             initial_data = model_to_dict(self)
+            if self.user is not None:
+                user = self.user
```
Comment (realsdx) [inline main/models.py:128]: how about just `self.user.is_active = False`.

**Comment by realsdx** [comment]
I am merging it anyways. Good work.

**Pull request merged**